addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -2720,7 +2720,7 @@ export class Order extends PosModel {
         return false;
     }
     is_paid() {
-        return this.get_due() <= 0 && this.check_paymentlines_rounding();
```

Comment:
i see that we already check if the rounding is valid
https://github.com/odoo/odoo/blob/36af138d231bffbfaff28d42bca972c1ddf172e2/addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.js#L245-L252
Perhaps we shouldn't even prevent the user from clicking on "Validate". This has the advantage that they will be able too hopefully understand what the problem is with the help of the error message.